sormas-ui/src/main/java/de/symeda/sormas/ui/campaign/campaigndata/CampaignFormDataEditForm.java

```diff
@@ -145,6 +150,23 @@ protected void addFields() {
 		}
 	}
 
+	private void hideAndFillJurisdictionFields() {
+		RegionReferenceDto defaultRegion = FacadeProvider.getRegionFacade().getDefaultInfrastructureReference();
+		cbRegion.setVisible(false);
+		cbRegion.setValue(defaultRegion);
+
+		cbDistrict.setVisible(false);
+		cbDistrict.setValue(FacadeProvider.getDistrictFacade().getDefaultInfrastructureReference());
+
+		cbCommunity.setVisible(false);
+		cbCommunity.setValue(FacadeProvider.getCommunityFacade().getDefaultInfrastructureReference());
+
+		if (FacadeProvider.getFeatureConfigurationFacade().isFeatureEnabled(FeatureType.INFRASTRUCTURE_TYPE_AREA)) {
+			cbArea.setVisible(false);
+			cbArea.setValue(FacadeProvider.getRegionFacade().getByUuid(defaultRegion.getUuid()).getArea());
```

Comment:
Will throw a NPE if there is no default region; there will be an error when trying to submit the form anyway in this case, but I'd rather not have an exception thrown here already